streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -151,6 +151,22 @@ public void shouldNotAllowToResetWhenIntermediateTopicAbsent() {
         Assert.assertEquals(1, exitCode);
     }
 
+    @Test
+    public void shouldNotAllowToResetWhenSpecifiedInternalTopicAbsent() {
```

Comment:
Just wondering, why put this test here instead of in `AbstractResetIntegrationTest`?

Reply:
Also, can we add a test like this but for the case where the topic does exist but just isn't a subset of inferred internal topics?

Reply:
It seems more natural to group it together with the other `shouldNotAllowToResetWhen...` tests. E.g. shouldNotAllowToResetWhenIntermediateTopicAbsent, shouldNotAllowToResetWhenInputTopicAbsent, etc.

Happy to shift it over to AbstractResetIntegrationTest.java

Reply:
Hah, maybe I should have asked why are all of those tests not also in `AbstractResetIntegrationTest`. Seems like almost everything that applies here would also be good to test in the SSL version of the test (which AFAICT is the only other one to extend the AbstractResetIntegrationTest).

But I'm ok with leaving it as is, and maybe we can just look into this as followup work unless there is a good reason for them to be where they are (which I can't think of but my imagination is not endless)